packages/larva-tokens/src/brands/variety.json

```diff
@@ -0,0 +1,60 @@
+{
+	"imports": [
+		"../base/aliases.json",
+		"../base/color.json",
+		"../base/font-family.json",
+		"../base/background-color.json"
+	],
+	"props": {
+		"BACKGROUND_COLOR_BRAND_PRIMARY": {
+			"value": "{!YELLOW_48}",
+			"type": "color",
+			"category": "background-color"
+		},
+		"BACKGROUND_COLOR_BRAND_ACCENT_A_100B": {
+			"value": "{!MIRAGE_14}",
+			"type": "color",
+			"category": "background-color"
+		},
+		"BORDER_COLOR_BRAND_PRIMARY": {
+			"value": "{!BLUE_61}",
+			"type": "color",
+			"category": "hr-color"
+		},
+		"COLOR_BRAND_PRIMARY": {
+			"value": "{!YELLOW_48}",
+			"type": "color",
+			"category": "text-color"
+		},
+		"COLOR_BRAND_SECONDARY": {
+			"value": "{!BLUE_61}",
+			"type": "color",
+			"category": "text-color"
+		},
+		"FONT_FAMILY_PRIMARY_FANCY": {
+			"value": "{!PARA_SUPREME}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY_FANCY": {
+			"value": "{!IBM_PLEX_SANS}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_ACCENT": {
+			"value": "{!GRAPHIK_COND}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BODY": {
+			"value": "{!IBM_PLEX_SERIF}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BASIC": {
```

Comment:
Anything that is a webfont needs to have a fallback - basic is intended to be a websafe font like Helvetica. If it is a webfont, it should be added as FONT_FAMILY_BASIC_FANCY with a fallback, FONT_FAMILY_BASIC.

Reply:
Also - can this be FONT_FAMILY_MONO, as above? The tokens between Variety and Variety VIP should be as similar as possible

Reply:
This is an overlook in my part `FONT_FAMILY_MONO` should be `FONT_FAMILY_ACCENT` for vip, which I'll update. Thanks for the catch!

These names are taken from the invision styleguides, where Variety and VIP share a naming convention, but differ in values.